packages/db/src/loaders/resources/contracts/index.ts

```diff
@@ -23,29 +23,31 @@ export function* generateContractsLoad(
   DataModel.IContract[],
   WorkspaceResponse<"contractsAdd", DataModel.IContractsAddPayload>
 > {
-  const contracts = loadableContracts.map(loadableContract => {
-    const {
-      contract: { contractName: name, abi: abiObject },
-      path: { sourceIndex, contractIndex },
-      bytecodes,
-      compilation
-    } = loadableContract;
+  const contracts = loadableContracts
+    .filter(({ contract }) => contract.bytecode.bytes !== "")
```

Comment:
probably worth a comment here about why this gets filtered out

Reply:
added!